internal/kms/kms.go

```diff
@@ -38,6 +39,16 @@ type KMS interface {
 	// CreateKey creates a new key at the KMS with the given key ID.
 	CreateKey(ctx context.Context, keyID string) error
 
+	// DeleteKey deletes a key at the KMS with the given key ID.
```

Comment:
Maybe we can split this into two KMS interfaces:
```
type KMS interface {
   // Here all methods implemented by KES and the static key KMS
}
```

```
type KES interface {
   // Here all KES-specific methods. 
} 
```

Later we can simply check whether `GlobalKMS` is a KES client via:
```
if kms, ok := GlobalKMS.(KES); ok {
   kms.ListKeys(ctx)
}
```